core/src/main/java/uk/gov/justice/services/core/eventbuffer/PassThroughEventBufferService.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.justice.services.core.eventbuffer;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import uk.gov.justice.services.event.buffer.api.EventBufferService;
+import uk.gov.justice.services.messaging.JsonEnvelope;
+
+import java.util.stream.Stream;
+
+import javax.annotation.Priority;
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Alternative;
+
+import static uk.gov.justice.services.messaging.logging.JsonEnvelopeLoggerHelper.toEnvelopeTraceString;
+
+/**
+ * Basic implementation of the EventBufferService passing the envelope through without any buffering
+ */
+@ApplicationScoped
+@Alternative
+@Priority(1)
+public class PassThroughEventBufferService implements EventBufferService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(EventBufferService.class);
+
+    @Override
+    public Stream<JsonEnvelope> currentOrderedEventsWith(final JsonEnvelope jsonEnvelope) {
+        LOGGER.trace("Message: %s passing through to dispatcher", toEnvelopeTraceString(jsonEnvelope));
```

Comment:
Trace logging doesn't log the whole envelope - on purpose. We decided to prevent the accidental exposure of large or sensitive data in the logs.